pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol_test.go

```diff
@@ -19,7 +19,7 @@ func setupTestEnv(t testing.TB) *OSSAccessControlService {
 
 	cfg := setting.NewCfg()
 	cfg.FeatureToggles = map[string]bool{"accesscontrol": true}
-	ac := ProvideService(cfg, &usageStatsMock{metricsFuncs: make([]usagestats.MetricsFunc, 0)})
+	ac := ProvideService(cfg, &usagestats.UsageStatsMock{MetricsFuncs: make([]usagestats.MetricsFunc, 0)})
```

Comment:
```suggestion
	ac := ProvideService(cfg, &usagestats.UsageStatsMock{T: t, MetricsFuncs: make([]usagestats.MetricsFunc, 0)})
```

Reply:
thanks, I think I need to remove `make([]usagestats.MetricsFunc, 0)` basically everywhere where we use this mock.

Reply:
yeah afaik it's not needed anywhere which will keep things nice and clean :)